brickadia/egg-brickadia.json

```diff
@@ -0,0 +1,100 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PANEL",
+    "meta": {
+        "version": "PLCN_v1",
+        "update_url": null
+    },
+    "exported_at": "2025-07-25T18:11:42+00:00",
+    "name": "Brickadia",
+    "author": "felinusfish@gmail.com",
+    "uuid": "3ee993b6-124a-4260-85fe-b622ebddded7",
+    "description": "Brickadia is a new multiplayer brick building sandbox game that gives you all the tools to create and play, your way. Build and invent with your friends, drive a car through space, create your own interactive experiences, or blow it all to smithereens!",
+    "tags": [],
+    "features": [],
+    "docker_images": {
+        "Debian": "ghcr.io\/parkervcp\/steamcmd:debian",
+        "Mono": "ghcr.io\/parkervcp\/yolks:mono_latest"
+    },
+    "file_denylist": [],
+    "startup": ".\/Brickadia\/Binaries\/Linux\/BrickadiaServer-Linux-Shipping -- -token={{BRICKADIA_TOKEN}} -- -port={{server.allocations.default.port}}",
```

Comment:
Are the -- needed? Why not just -token -port. Also on startup please use {{SERVER_PORT}} as the way you did it is only for the file parser.